### PR TITLE
Dropping support for python 3.7

### PR DESCRIPTION
**Pull request by levmckinney**
Drops support for python version 3.7.

### CODE REVIEW

**Comment by AdamGleave** [comment]
Seems reasonable. The PyPI automation failing is not your fault and is tracked in https://github.com/HumanCompatibleAI/imitation/issues/500

File: setup.py [hunk 1]
```diff
@@ -161,7 +161,6 @@ def run(self):
         "License :: OSI Approved :: MIT License",
         "Programming Language :: Python",
         "Programming Language :: Python :: 3",
```
Comment (levmckinney) [inline setup.py:163]: Are we only supporting 3.8 or do we also support 3.9 and 3.10?

Comment (levmckinney) [reply setup.py:163]: `stable-baselines` says it to supports up to 3.9.

**Review by AdamGleave — APPROVED** [review]
LGTM.

We support later Python versions, I've run it with 3.10 successfully, we should add it to `setup.py`.

We're not testing other versions in CI but it'd be pretty weird for later versions to break, Python aims to be backward compatible, the main issues I've run into with later Python versions has been sometimes binary dependencies don't exist for a particular wheel (e.g. TF 1.15 only goes up to 3.7 support).

**Pull request merged**